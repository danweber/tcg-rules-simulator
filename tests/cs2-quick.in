VERSION:2
# [Main] Delete all of your opponent's Monsters with the lowest level.
# p1 has three Lv.4, and four Lv.5, and three of those Lv.5 are phoenix.
FIELD1:Four1,CS1-06 Four2,CS1-06 Four3,CS1-07 Five1,CS1-Phoenix Five2,CS1-Phoenix Five3,CS1-Phoenix, Five4,CS1-Solid
EGGS2:CS2-01
HAND2:CS2-Quick CS2-Quick
DECK1:CS1-02
GAME:P2 T3 HATCHING 9
CMD:HATCH
DUMP:EGGZONE,TRASH

ECHO: -- Use first quick slash, kill the level 4s
CMD:PLAY CS2-Quick
STEP:AUTOANSWER:CAPTURE
DUMP:TRASH

ECHO: -- Use second quick slash, kill the level 5s, Phoenix all come back
CMD:PLAY CS2-Quick
EFFECT-TREE
DUMP

ECHO: -- pick first pheonix
STEP:TO:0
EFFECT-TREE
DUMP

ECHO: -- pick second pheonix... all effects insta-resolve now
STEP:TO:0
EFFECT-TREE
DUMP

ECHO: ----- NOTHING LEFT
STEP:AUTOANSWER:CAPTURE
DUMP
