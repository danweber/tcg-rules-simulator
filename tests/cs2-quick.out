FIELD1:Four1,CS1-Sneakycat Four2,CS1-Sneakycat Four3,CS1-Slipperyshield Five1,CS1-Phoenixdefender Five2,CS1-Phoenixdefender Five3,CS1-Phoenixdefender Five4,CS1-Solidsteel
FIELD2:
TRASH1:
TRASH2:
EGGZONE1:
EGGZONE2:Xxx,CS2-Doggiedagger
GAME:P2 T3 MAIN 9
ECHO: -- Use first quick slash, kill the level 4s
FIELD1:Five1,CS1-Phoenixdefender Five2,CS1-Phoenixdefender Five3,CS1-Phoenixdefender Five4,CS1-Solidsteel
FIELD2:
TRASH1:CS1-Sneakycat CS1-Sneakycat CS1-Slipperyshield
TRASH2:CS2-Quickslash
GAME:P2 T3 MAIN 3
ECHO: -- Use second quick slash, kill the level 5s, Phoenix all come back
EFFECT-TREE:DSE DEPTH 1 DONE:use Quick Slash
EFFECT-TREE:RES DEPTH 2 DONE:Quick Slash
EFFECT-TREE:RES DEPTH 3 Phoenix Defender, Phoenix Defender, Phoenix Defender
FIELD1:
FIELD2:
GAME:P2 T3 MAIN -3
ECHO: -- pick first pheonix
EFFECT-TREE:DSE DEPTH 1 DONE:use Quick Slash
EFFECT-TREE:RES DEPTH 2 DONE:Quick Slash
EFFECT-TREE:RES DEPTH 3 Phoenix Defender, Phoenix Defender, DONE:Phoenix Defender
FIELD1:I9,CS1-Phoenixdefender
FIELD2:
GAME:P2 T3 MAIN -3
ECHO: -- pick second pheonix... all effects insta-resolve now
EFFECT-TREE:NOTHING
FIELD1:I9,CS1-Phoenixdefender I10,CS1-Phoenixdefender I11,CS1-Phoenixdefender
FIELD2:
GAME:P1 T4 HATCHING 3
ECHO: ----- NOTHING LEFT
FIELD1:I9,CS1-Phoenixdefender I10,CS1-Phoenixdefender I11,CS1-Phoenixdefender
FIELD2:
GAME:P1 T4 HATCHING 3
